setup.py

```diff
@@ -10,10 +10,7 @@
       classifiers=['Programming Language :: Python :: 3 :: Only'],
       py_modules=['tap_zendesk'],
       install_requires=[
-          # Currently this refers to our fork. This will only work if we
-          # specifically install the correct version of our fork before
-          # trying to install this package.
-          'singer-python==5.7.*',
```

Comment:
Can we keep this comment since it still applies? Offline we tried `singer-python==5.12.2` which was the official version, but this time around we're using our fork

Reply:
yup